src/utils/index.js

```diff
@@ -123,13 +123,20 @@ module.exports.isMobile = function () {
       check = true;
     }
   })(navigator.userAgent || navigator.vendor || window.opera);
+  if (isGearVR()) {
```

Comment:
I think, this should be inside the function above. Below the If isIOS statement.